### ISSUE TITLE
NullPointerException when using Tab Bridge 2.0.4

### ISSUE THREAD

**Issue by daffychuy** [NEZNAMY/TAB-Bridge#8]
From what I can see in-game nothing is broken, Tab is displaying correctly and no glitches from what I can see, just this error is being thrown in the console.

Configuration:
- BungeeCord v1.19.1 (1.19-R0.1-SNAPSHOT:78ca16d:1656)
- Purpur v1.19.1 (build 1747)
- TAB (in Bungeecord) v3.1.3
- TAB-Bridge v2.0.4

The following errors are being thrown occasionally but multiple times if this error is present
````[08:36:42 WARN]: [TAB-Bridge] Plugin TAB-Bridge v2.0.4 generated an exception while executing task 32
java.lang.NullPointerException: Cannot invoke "me.neznamy.tab.bridge.bukkit.features.unlimitedtags.ArmorStandManager.respawn()" because the return value of "me.neznamy.tab.bridge.bukkit.features.unlimitedtags.BridgeNameTagX.getArmorStandManager(me.neznamy.tab.bridge.shared.BridgePlayer)" is null
        at me.neznamy.tab.bridge.bukkit.features.unlimitedtags.VehicleRefresher.refresh(VehicleRefresher.java:83) ~[TAB-Bridge v2.0.4.jar:?]
        at me.neznamy.tab.bridge.bukkit.features.unlimitedtags.VehicleRefresher.lambda$new$0(VehicleRefresher.java:43) ~[TAB-Bridge v2.0.4.jar:?]
        at org.bukkit.craftbukkit.v1_19_R1.scheduler.CraftTask.run(CraftTask.java:101) ~[purpur-1.19.1.jar:git-Purpur-1747]
        at org.bukkit.craftbukkit.v1_19_R1.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:57) ~[purpur-1.19.1.jar:git-Purpur-1747]
        at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22) ~[purpur-1.19.1.jar:?]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
        at java.lang.Thread.run(Thread.java:889) ~[?:?]```

**Comment by daffychuy** [comment]
There's actually another error that was just thrown, I'll just append it here as it seems to be related
```
>....2022-08-03 00:51:57,620 Craft Scheduler Thread - 890 - TAB-Bridge ERROR Recursive call to appender Async
[00:51:57 WARN]: [TAB-Bridge] Plugin TAB-Bridge v2.0.4 generated an exception while executing task 26
java.lang.NullPointerException: Cannot invoke "me.neznamy.tab.bridge.bukkit.features.unlimitedtags.ArmorStandManager.respawn()" because the return value of "me.neznamy.tab.bridge.bukkit.features.unlimitedtags.BridgeNameTagX.getArmorStandManager(me.neznamy.tab.bridge.shared.BridgePlayer)" is null
        at me.neznamy.tab.bridge.bukkit.features.unlimitedtags.VehicleRefresher.refresh(VehicleRefresher.java:83) ~[TAB-Bridge v2.0.4.jar:?]
        at me.neznamy.tab.bridge.bukkit.features.unlimitedtags.VehicleRefresher.lambda$new$0(VehicleRefresher.java:43) ~[TAB-Bridge v2.0.4.jar:?]
        at org.bukkit.craftbukkit.v1_19_R1.scheduler.CraftTask.run(CraftTask.java:101) ~[purpur-1.19.1.jar:git-Purpur-1747]
        at org.bukkit.craftbukkit.v1_19_R1.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:57) ~[purpur-1.19.1.jar:git-Purpur-1747]
        at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22) ~[purpur-1.19.1.jar:?]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136) ~[?:?]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635) ~[?:?]
        at java.lang.Thread.run(Thread.java:889) ~[?:?]
```

**Comment by NEZNAMY** [comment]
Do you have unlimited nametag mode enabled?

**Comment by NEZNAMY** [comment]
Fixed in 2.0.5 if you do.
